src/domain/WarpGraph.js

```
@@ -1371,11 +1446,268 @@ export default class WarpGraph {
+      }
+    };
+
+    // Reuse subscription infrastructure
+    const subscription = this.subscribe({ onChange: filteredOnChange, onError });
+
+    // Polling: periodically check frontier and auto-materialize if changed
+    let pollIntervalId = null;
+    let pollInFlight = false;
+    if (poll) {
+      pollIntervalId = setInterval(async () => {
+        if (pollInFlight) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)